contractsSol5/KyberNetwork.sol

```diff
@@ -33,9 +34,10 @@ contract KyberNetwork is Withdrawable2, Utils4, IKyberNetwork, ReentrancyGuard {
     uint  internal constant DEFAULT_NETWORK_FEE_BPS = 25;    // till we read value from DAO
     uint  internal constant MAX_APPROVED_PROXIES = 2;        // limit number of proxies that can trade here.
 
-    IKyberFeeHandler[]      internal feeHandler;
-    IKyberDAO[]             internal kyberDAO;
-    IKyberMatchingEngine[]  internal matchingEngine;
+    IKyberFeeHandler        public   feeHandler;
+    IKyberDAO               public   kyberDAO;
+    IKyberMatchingEngine    public   matchingEngine;
+    IKyberStorage           public   kyberStorage;
```

Comment:
where is kyberStorage set?
should set it in ctor.
and for now can't change it.

Reply:
added `setKyberStorage` method